node_modules

```diff
@@ -0,0 +1 @@
+/webapp-frontend-deps/node_modules
```

Comment:
We should get rid of this as part of #998.

Reply:
Right, and you should actually remove this file from your branch please. :) (It's conflicting with my local files. )